content/W2015/W15-event-research-showcase.md

```diff
@@ -1,7 +1,7 @@
 Title: Research Showcase and Mixer
 Date: 2015-02-04 17:30
 Category: Events
-Tags: talks, academia, social
+Tags: talks, academic, social
```

Comment:
1. I am suggesting we come to a group decision on that. @wics-uw/contributors do you have any thoughts?
2. We can pair on this, just get on IRC and I'll walk you through it.